File: Python/Civitas.py
from Human import Human

class Civitas(Human):
    # civitas adalah child dari Human
    
    # atribut privat
    __univ = ""
    __email = ""
    
    # constructor
    def __init__(self, nik="", name="", gender="", univ="", email=""):
        super().__init__(nik, name, gender)
        self.__univ = univ
        self.__email = email
    
    # getter setter
    def setUniv(self, univ):
        self.__univ = univ
    
    def getUniv(self):
        return self.__univ
    
    def setEmail(self, email):
        self.__email = email
    
    def getEmail(self):
        return self.__email

File: Java/Civitas.java
public class Civitas extends Human {

    // atribut privat
    private String univ;
    private String email;

    // constructor
    public Civitas(String univ, String email) {
        this.univ = univ;
        this.email = email;
    }

    public Civitas() {
    }

    // getter setter
    public String getUniv() {
        return this.univ;
    }

    public void setUniv(String value) {
        this.univ = value;
    }

    public String getEmail() {
        return this.email;
    }

    public void setEmail(String value) {
        this.email = value;
    }
}
